src/Symfony/Component/Notifier/Tests/Event/SentMessageEventTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Symfony\Component\Notifier\Tests\Event;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Mime\RawMessage;
+use Symfony\Component\Notifier\Event\SentMessageEvent;
+use Symfony\Component\Notifier\Message\ChatMessage;
+use Symfony\Component\Notifier\Message\EmailMessage;
+use Symfony\Component\Notifier\Message\SentMessage;
+use Symfony\Component\Notifier\Message\SmsMessage;
+
+/**
+ * @author Smaïne Milianni <smaine.milianni@gmail.com>
+ */
+class SentMessageEventTest extends TestCase
+{
+    /**
+     * @dataProvider dataprovider
+     */
+    public function testConstruct(SentMessage $message, SentMessageEvent $event)
+    {
+        $this->assertEquals($event, new SentMessageEvent($message));
+    }
+
+    /**
+     * @dataProvider dataprovider
+     */
+    public function testGetMessage(SentMessage $message, SentMessageEvent $event)
+    {
+        $this->assertEquals($message, $event->getMessage());
+    }
+
+    public function dataprovider(): iterable
```

Comment:
I would rename the provider:

```suggestion
    public function provideMessages(): iterable
```

Or messagesProvider